maps/_common/mapsystem/map.dm

```diff
@@ -168,9 +168,18 @@
 
 	var/datum/space_sector/sector = SSatlas.current_sector
 	var/list/possible_exoplanets = sector.possible_exoplanets
+	var/list/guaranteed_exoplanets = sector.guaranteed_exoplanets
+
+	if(length(guaranteed_exoplanets))
+		for(var/j = 0, j < min(guaranteed_exoplanets.len, num_exoplanets), j++)
```

Comment:
why not just loop through the list of exoplanets here? if the exoplanets are guaranteed, i dunno if they should be limited by num_exoplanets

Reply:
The intent was to avoid overly bloated sectors, especially in the case of Haneunim where Konyang is very much the star of the show.